### ISSUE TITLE
Issue when running sourcery from binary

### ISSUE THREAD

**Issue by ugoArangino** [krzysztofzablocki/Sourcery#1202]
Version: 2.1.0

> SourceryJS/resource_bundle_accessor.swift:11: Fatal error: could not load resource bundle: from /Users/…/Documents/ios/bin/Sourcery_SourceryJS.bundle or /Users/art-divin/Documents/Projects/Sourcery-master/build/arm64-apple-macosx/release/Sourcery_SourceryJS.bundle

**Comment by ugoArangino** [comment]
@art-divin Looks like this PR has introduced this: https://github.com/krzysztofzablocki/Sourcery/pull/1188

**Comment by JohnEstropia** [comment]
We're seeing this issue as well

**Comment by art-divin** [comment]
👋🏻  Hello

Thank you for reporting @ugoArangino and @JohnEstropia ,

I'll need some time to figure this out. Could you please fallback to using 2.0.2 or, if performance is not that big of a deal, 2.0.3?

I'll keep you posted ⏲️

**Comment by alpha-moisol** [comment]
Same issue here

**Comment by piwo33** [comment]
@art-divin how exactly should we rollback to version 2.0.2 / 2.0.3 with `brew`?

**Comment by art-divin** [comment]
@piwo33 thank you, makes sense, very cumbersome with Brew.

Testing on CI #1203 , if green I'll merge it and try to release 2.1.1 hotfix today.

**Comment by dafurman** [comment]
If anyone using homebrew wants a workaround, run this from your download directory to downgrade down to 2.0.3

`curl -L https://github.com/krzysztofzablocki/Sourcery/releases/download/2.0.3/sourcery-2.0.3.zip --output sourcery-2.0.3.zip && unzip -d sourcery-2.0.3 sourcery-2.0.3.zip && sudo cp sourcery-2.0.3/bin/sourcery /opt/homebrew/bin/sourcery`

**Comment by art-divin** [comment]
Update: I figured a way to workaround the issue.

## Context

For `swift test` to work, a different way of searching for `ejs` template on macOS was implemented according to "best practices" due to issues with SPM (more info here).
But the problem with this is that when running Sourcery from CLI, it is packaged differently for `release` mode, and so, it cannot find the required resources like for `swift test`.

## Workaround

1. Revert changes for before the migration to `swift test`
2. Create a separate version of implementation that works with `swift test` and put it under `#if DEBUG` because `swift test` runs tests in DEBUG, and Sourcery CLI is distributed with `-c release` compilation mode.

Waiting for CI ⏲️

**Comment by art-divin** [comment]
I have re-opened this issue until the new release is out ⏲️

**Comment by diegoccastano** [comment]
> curl -L https://github.com/krzysztofzablocki/Sourcery/releases/download/2.0.3/sourcery-2.0.3.zip --output sourcery-2.0.3.zip && unzip -d sourcery-2.0.3 sourcery-2.0.3.zip && sudo cp sourcery-2.0.3/bin/sourcery /opt/homebrew/bin/sourcery

In a similar way, I also solved the problem by doing `brew install sourcery`, which will install the most current version, and then I give the `brew info sourcery` command to find out where the installation was done, so I find out where the binary folder is, and I overwrite it with this command above that I will repeat here to make it easier: 

```shellscript
curl -L https://github.com/krzysztofzablocki/Sourcery/releases/download/2.0.3/sourcery-2.0.3.zip --output sourcery -2.0.3.zip && unzip -d sourcery-2.0.3 sourcery-2.0.3.zip && sudo cp sourcery-2.0.3/bin/sourcery /usr/local/Cellar/sourcery/2.1.0/bin/sourcery
```

In this case, `sorcery-2.0.3/bin/sorcery /usr/local/Cellar/sourcery/2.1.0/bin/sourcery` is the copy destination, that's what I discovered with the help of `brew info`. The version overwritten by version 2.0.3 is 2.1.0 (most current). If you want to install other versions, just go to releases in the Sourcery repository on GitHub.

**Comment by art-divin** [comment]
👋🏻  Update:

[2.1.1 release was published](https://github.com/krzysztofzablocki/Sourcery/releases/tag/2.1.1). Please see if you have any issues, thank you 🙏🏻

**Comment by philprime** [comment]
@art-divin when is the release 2.1.1 going to be available on Homebrew?

**Comment by art-divin** [comment]
👋🏻  @philprime, thank you for your question 💯 

I think it happens automatically, for 2.1.0 I did exactly nothing to make it available there. ⏲️

**Comment by mol-ideals** [comment]
> @art-divin when is the release 2.1.1 going to be available on Homebrew?




> 👋🏻 @philprime, thank you for your question 💯
> 
> I think it happens automatically, for 2.1.0 I did exactly nothing to make it available there. ⏲️

It seems like the PR actually needs to be done manually. Fortunately, someone has already done it: https://github.com/Homebrew/homebrew-core/pull/143894

**Comment by chenrui333** [comment]
> > @art-divin when is the release 2.1.1 going to be available on Homebrew?
> 
> > 👋🏻 @philprime, thank you for your question 💯
> > I think it happens automatically, for 2.1.0 I did exactly nothing to make it available there. ⏲️
> 
> It seems like the PR actually needs to be done manually. Fortunately, someone has already done it: [Homebrew/homebrew-core#143894](https://github.com/Homebrew/homebrew-core/pull/143894)

just raise an issue on the homebrew release part, https://github.com/krzysztofzablocki/Sourcery/issues/1205 (should be pretty straightforward for maintainer to pick it up.)

**Comment by art-divin** [comment]
Thank you all for your input and for raising this issue.

Homebrew MR was merged some 45 minutes ago, and so, please check if the 2.1.1 would work in your environments, I do not have much to test it against, except very basic setups.

⏳

**Comment by ericklborges** [comment]
It is already available, I just upgraded to `2.1.1`, and it worked just fine!

**Comment by art-divin** [comment]
Thanks for your feedback @ericklborges , it was a tough release, my internet barely made it 😓 😄
